hack/generator/pkg/astmodel/file_definition.go

```diff
@@ -204,3 +208,33 @@ func (file FileDefinition) SaveToFile(filePath string) error {
 
 	return file.SaveToWriter(filePath, f)
 }
+
+func (file FileDefinition) AddBlankLinesBeforeComments(buffer bytes.Buffer) bytes.Buffer {
```

Comment:
Aren't function comments required for public funcs?

Reply:
Although, I believe we're using golangci-lint rather than golint which I think defaults to not include this check. The ASO repo doesn't use golangci-lint from what I can see so we need to be aware there may be situations like this where suddenly our code is throwing warnings once we migrate it to ASO. We could of course enable golangci-lint in ASO as well...

Reply:
This method doesn't need to be exported, so renamed it to make it internal. Added a documentation comment anyway to clarify intent.